astronomer/providers/amazon/aws/operators/redshift_data.py

```diff
@@ -0,0 +1,67 @@
+from typing import Any
+
+from airflow.exceptions import AirflowException
+from airflow.providers.amazon.aws.operators.redshift_data import RedshiftDataOperator
+from airflow.utils.context import Context
+
+from astronomer.providers.amazon.aws.hooks.redshift_data import RedshiftDataHook
+from astronomer.providers.amazon.aws.triggers.redshift_data import RedshiftDataTrigger
+
+
+class RedshiftDataOperatorAsync(RedshiftDataOperator):
+    """
+    Executes SQL Statements against an Amazon Redshift cluster.
```

Comment:
What about mentioning a bit about behaviour also here like in case of multiple if some query fails what we are doing?